services/pane.js

```diff
@@ -91,36 +90,40 @@ function createPublishMessages(res) {
  * @returns {Element}
  */
 function createPublishActions(res) {
-  var today = moment().format('YYYY-MM-DD'),
+  var actions = dom.find('.publish-actions'),
```

Comment:
hmm, actually, I guess all of the template are within `kiln-pane-template-elements` so that should be shared?